Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
Which further enforces the fact this revert is an incorrect fix to a problem. If two other JITs can work fine, and the outlying one does not, it's a problem with the JIT, not the merged code.